air-script/tests/selectors/selectors_with_evaluators.air

```diff
@@ -1,33 +1,33 @@
 def SelectorsAir
 
 ev next_is_zero([clk]) {
-    enf clk' = 0
+    enf clk' = 0;
 }
 
 ev is_unchanged([clk, s]) {
-    enf clk' = clk when s
+    enf clk' = clk when s;
 }
 
 ev next_is_one([clk]) {
-    enf clk' = 1
+    enf clk' = 1;
 }
 
 trace_columns {
-    main: [s[3], clk]
+    main: [s[3], clk],
 }
 
 public_inputs {
-    stack_inputs: [16]
+    stack_inputs: [16],
 }
 
 boundary_constraints {
-    enf clk.first = 0
+    enf clk.first = 0;
 }
 
 integrity_constraints {
-    enf next_is_zero([clk]) when s[0] & !s[1]
+    enf next_is_zero([clk]) when s[0] & !s[1];
     enf match {
         case s[1] & s[2]: is_unchanged([clk, s[0]])
         case !s[1] & !s[2]: next_is_one([clk])
-    }
+    };
```

Comment:
Should we add commas to the end of each `case` clause? For example, something like this:
```rust
enf match {
    case s[1] & s[2]: is_unchanged([clk, s[0]]),
    case !s[1] & !s[2]: next_is_one([clk]),
};
```

Reply:
I was planning to add that in another PR but let me know if you want in this one?

Reply:
A follow-up PR is fine! I'll merge this one now then.